app/models/item.rb

```diff
@@ -1,3 +1,3 @@
 class Item < ActiveRecord::Base
-    has_many :facilities, through: :facility_items
+    has_one :facilities, :foreign_key => "facility_id"
```

Comment:
This should be singular (and use the updated hash syntax):
```suggestion
    has_one :facility, foreign_key: "facility_id"
```

Reply:
Changed "facilities" to "facility" and used hash syntax in item and procedure models.